static/js/show-task.js

```diff
@@ -39,20 +41,56 @@ function saveTask() {
     }});
 }
 
+function rememberState() {
+	// Finding how many number of rows are being displayed so that we can make it same after refresh
+        var dropdownElements = $('.page-list .dropdown-menu .dropdown-item');
+        var activeDropdownElement = $('.page-list .dropdown-menu .dropdown-item.active')[0];
+        for(var i=0; i < dropdownElements.length; i++)
+        {
+            if (dropdownElements[i] === activeDropdownElement) {
+                activeDropdownElementIndex = i;
+            }
+        }
+
+        // Finding which page is active so that we can make it active again after refresh
+        var paginationElements = $('.pagination .page-item .page-link');
+        var activeElement = $('.pagination .page-item.active .page-link')[0];
+        for(var i=0; i < paginationElements.length; i++)
+        {
+            if (paginationElements[i] === activeElement) {
+                activeElementIndex = i;
+            }
+        }
+}
```

Comment:
This one could possibly also use `index()` if we cut the selector at the `.active` part.

Reply:
Done!